api/tests/integration/environments/identities/test_integration_identities.py

```diff
@@ -149,12 +149,19 @@ def test_get_feature_states_for_identity_only_makes_one_query_to_get_mv_feature_
             ],
         )
 
-    # When we make a request to get the flags for the identity, 11 queries are made
+    base_number_of_queries = 4
+    number_of_integrations = len(
+        list(
+            filter(lambda app: app.startswith("integrations."), settings.INSTALLED_APPS)
+        )
+    )
+
+    # When we make a request to get the flags for the identity, 12 queries are made
     # (although 4 of these are made in a separate thread)
```

Comment:
I don't think this is true anymore?

Reply:
Hmm - maybe it's not 4/8 any more - but this test passes?